projects/plugins/jetpack/tests/php/modules/masterbar/test-class-admin-menu.php

```diff
@@ -253,12 +254,11 @@ public function test_add_custom_post_type_menu() {
 	 * @covers ::add_comments_menu
 	 */
 	public function test_add_comments_menu() {
-		global $menu, $submenu;
+		global $menu;
 
 		static::$admin_menu->add_comments_menu();
 
 		$this->assertSame( 'https://wordpress.com/comments/all/' . static::$domain, $menu[25][2] );
-		$this->assertEmpty( $submenu['edit-comments.php'] );
```

Comment:
Instead of removing this assert, can we change it so we check there are no visible submenus?

Reply:
Yup